frontend/src/App.jsx

```diff
@@ -1,5 +1,5 @@
 import { Route, Routes } from "react-router-dom";
-import SharedLayout from "components/SharedLayout/SharedLayout";
+import SharedLayout from "components/SharedLayout/index";
```

Comment:
gdy robisz index.jsx to nie musisz pisać index na końcu moze być
"components/SharedLayout"